apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-redirect.tsx

```diff
@@ -0,0 +1,19 @@
+import { FormLabel } from '@/components/primitives/form/form';
+import { URLInput } from '../../url-input';
+import { urlTargetTypes } from '@/utils/url';
+
+export const InAppRedirect = () => {
+  return (
+    <div className="flex flex-col gap-1">
+      <FormLabel hint="">Redirect URL</FormLabel>
+      <URLInput
+        options={urlTargetTypes}
+        placeholder="/tasks/{{taskid}}"
+        size="md"
+        asEditor
+        urlKey="redirect.url"
+        targetKey="redirect.target"
```

Comment:
Maybe name these `urlFieldName` and `targetFieldName`, or a `fieldNames` prop that has both? Could be a nice convention for components with multiple inputs.

Reply:
Nice, I like it, I will change that.